### PR TITLE
Polkadot support added for GetAndFundTestUsers() / SendFunds()

### PR DESCRIPTION
**Pull request by misko9**
Added explorer links for each node in output for monitoring execution. 
Overwrite session_length_in_blocks parameter to 10 in relay chain. With the current default as 100, this decreases setup time by 9 minutes.

### CODE REVIEW

File: chain/polkadot/polkadot_chain.go [hunk 1]
```diff
@@ -700,7 +703,19 @@ func (c *PolkadotChain) BuildRelayerWallet(ctx context.Context, keyName string)
 // SendFunds sends funds to a wallet from a user account.
 // Implements Chain interface.
 func (c *PolkadotChain) SendFunds(ctx context.Context, keyName string, amount ibc.WalletAmount) error {
-	panic("[SendFunds] not implemented yet")
+	// If denom == polkadot denom, it is a relay chain tx, else parachain tx
+	if(amount.Denom == c.cfg.Denom) {
```
Comment (DavidNix) [inline chain/polkadot/polkadot_chain.go:707]: nit,style: This line (and some other if statements) are not `go fmt`ed. The parens are also unnecessary. 

The following is idiomatic. 

```go
if amount.Denom == c.cfg.Demon {
```

I am disappointed CI is not catching this `go fmt` error. 

I am curious what running `go fmt ./...` does?

File: chain/polkadot/polkadot_chain.go [hunk 2]
```diff
@@ -718,29 +733,12 @@ func (c *PolkadotChain) SendIBCTransfer(
...
-
-	var accountInfo AccountInfo
-	ok, err := c.RelayChainNodes[0].api.RPC.State.GetStorageLatest(key, &accountInfo)
-	if err != nil {
-		return -4, err
-	}
-	if !ok {
-		return -5, nil
+	// If denom == polkadot denom, it is a relay chain query, else parachain query
+	if(denom == c.cfg.Denom) {
```
Comment (DavidNix) [inline chain/polkadot/polkadot_chain.go:737]: nit: See previous comment. Not `go fmt`ed.

**Review by agouin — APPROVED** [review]
Minor nits, otherwise LGTM

**Review by DavidNix — APPROVED** [review]

File: chain/polkadot/relay_chain_node.go [hunk 3]
```diff
@@ -311,3 +316,121 @@ func (p *RelayChainNode) Exec(ctx context.Context, cmd []string, env []string) d
 	}
 	return job.Run(ctx, cmd, opts)
 }
+
+// SendFunds sends funds to a wallet from a user account.
+// Implements Chain interface.
+func (p *RelayChainNode) SendFunds(ctx context.Context, keyName string, amount ibc.WalletAmount) error {
```
[the remark this answers is not in the document]
Comment (misko9) [reply chain/polkadot/relay_chain_node.go:320]: Created helpers for SendFunds and GetBalance

File: chain/polkadot/polkadot_chain.go (hunk 1 above)
Comment (misko9) [reply chain/polkadot/polkadot_chain.go]: Ran go `fmt ./...` and included the proper formatting in the last commit

**Pull request merged**